util/version/build.rs

```diff
@@ -33,5 +33,5 @@ fn main() {
 		pub fn rustc_version() -> &'static str {{
 			\"{}\"
 		}}
-	", rustc_version::version()).as_bytes()).unwrap();
+	", rustc_version::version().unwrap()).as_bytes()).unwrap();
```

Comment:
Why is it safe?

Reply:
it's not. neither of the `unwraps` here is safe. I will fix it